ui/extensions/hello/config/rollup.config.js

```diff
@@ -23,6 +23,8 @@ export default defineConfig([
       }),
       babel({
         presets: ["@babel/preset-react"],
+        babelHelpers: 'bundled',
+        exclude: 'node_modules/**'
```

Comment:
We might want to make these changes to the React blueprint to suppress warning messages.